src/state/stake/useFarmsPortfolio.ts

```diff
@@ -184,9 +184,10 @@ export function useFarmsPortfolio(farmIds?: number[]): Result | null {
   }
 
   const totalStaked = activeFarms.map((_, index) => {
-    if (stakingInfoData?.[index] == null) {
-      return BIG_INT_ZERO
+    if (!stakingInfoData?.[index] || stakingInfoData?.[index]?.totalStaked === 0) {
```

Comment:
The optional chaining operator (`?.`) isn't needed for the first part because `stakingInfoData` should always be nonnull; additionally, the first part covers you for for the second part, so if you wanted to keep it you can do:

```
if (!stakingInfoData[index] || stakingInfoData[index].totalStaked === 0) {
```

Or you can use the optional chaining operator with the nullish coalescing operator:

```
if (stakingInfoData?.[index]?.totalStaked ?? 0) !== 0) {
```

Reply:
ty! 👌🏻